linkis-commons/linkis-common/src/main/java/org/apache/linkis/common/io/FsPath.java

```diff
@@ -273,6 +271,10 @@ public String getSchemaPath() {
     if (WINDOWS && !"hdfs".equals(getFsType())) {
       return getFsType() + "://" + uri.getAuthority() + uri.getPath();
     }
+
+    if (uri.getPath().startsWith("/")) {
+      return getFsType() + "://" + uri.getPath().substring(1);
```

Comment:
The path of hdfs should be hdfs:///tmp

Reply:
Thank for your advice.I have fixed it, and have tested read/write operation on hdfs filesystem and OSS filesystem. Please have a further view thanks.

Reply:
> Thanks for your advice.I have fixed it, and have tested read/write operation on hdfs filesystem and OSS filesystem. Please have a further view thanks.

Thank you！